internal/mode/static/nginx/config/servers.go

```diff
@@ -294,6 +294,7 @@ func updateLocationsForFilters(
 
 	rewrites := createRewritesValForRewriteFilter(filters.RequestURLRewrite, path)
 	proxySetHeaders := generateProxySetHeaders(&matchRule.Filters, grpc)
+	responseHeaders := generateResponseHeaders(&matchRule.Filters)
```

Comment:
Per the comments on https://github.com/nginxinc/nginx-gateway-fabric/issues/1766, I wonder if we also add support for GRPCRoute for this filter? Or we can create a new ticket to be done as a followup to finish off the last few pieces. https://github.com/nginxinc/nginx-gateway-fabric/pull/1909 probably does a lot of the work up front relating to the common code.

Reply:
I can create a new ticket under the GRPC Route task to support response headers for that route as well.